examples/react-native-cli/metro.config.js

```diff
@@ -10,8 +10,14 @@ const root = path.resolve(__dirname, '../..');
  * @type {import('metro-config').MetroConfig}
  */
 const config = {
-  watchFolders: [root],
   resetCache: true,
+  watchFolders: [workspace],
+  resolver: {
+    nodeModulesPaths: [
+      path.resolve(__dirname, 'node_modules'),
+      path.resolve(workspace, 'node_modules'),
+    ]
+  },
```

Comment:
🙋‍♂️ can you add an explanation on why this is needed please :)

Reply:
I was getting error, like this

```
error: Error: Unable to resolve module @babel/runtime/helpers/interopRequireDefault from /Users/georgepoladashvili/WebstormProjects/rechunk/src/react-native/index.ts: @babel/runtime/helpers/interopRequireDefault could not be found within the project or in these directories:
```